bundles/org.openhab.ui.basic/gulpfile.js

```diff
@@ -3,17 +3,21 @@
 
 	var
 		gulp = require("gulp"),
+		replace = require('gulp-replace'),
 		sass = require('gulp-sass')(require('node-sass')),
 		uglify = require("gulp-uglify"),
 		eslint = require("gulp-eslint");
 
 	var
 		sources = {
 			js: [ "web-src/smarthome.js", "web-src/static.js", "web-src/settings.js" ],
-			sass: "web-src/smarthome.scss"
+			sass: "web-src/smarthome.scss",
+			html: {
+				all: "snippets-src/**",
```

Comment:
** ?
Maybe *.html ?

Reply:
ditto here, we don't want to confuse someone in the far future adding non .html and wondering why it didn't copy across. This way it's more predictable.

What are you trying to avoid by being more specific?

Reply:
I've renamed `html` to `snippets`. Perhaps that helps?

Reply:
What I do not understand is why you put a doubled *.

Reply:
It will cover subdirectories in case they're added in the future, to avoid surprises.

Reply:
Thank you for the explanation